app/controllers/v0/sign_in_controller.rb

```diff
@@ -96,12 +96,12 @@ def token
       response_body = SignIn::TokenResponseGenerator.new(params: token_params,
                                                          cookies: token_cookies,
                                                          request_attributes:).perform
-      sign_in_logger.info('token')
+      sign_in_logger.info('token', { grant_type: token_params[:grant_type] })
```

Comment:
Do we need this when we already have the grant_type logged in the `Completed #token` logs?